extension/content.js

```diff
@@ -263,6 +263,25 @@ function addPatchDiffLinks() {
 	`);
 }
 
+// Prompt user to confirm erasing a comment with the Cancel button
+$(document).on('click', function (event) {
+	// Check event.target instead of using a delegate, because Sprint doesn't support them
```

Comment:
@sindresorhus I meant if we create a jQuery build with very few modules and then we need another method which wasn't included initially we'll still need to update the jQuery build.